phpBB/includes/functions_compatibility.php

```diff
@@ -162,7 +162,7 @@ function cache_moderators()
 /**
 * Removes moderators and administrators from foe lists.
 *
-* @deprecated 3.1.0 (To be removed: 3.3.0)
```

Comment:
I guess the reasoning for this change is to not cause BC breaks in the 3.x line?